lib/rails_live_reload/config.rb

```diff
@@ -33,6 +37,11 @@ def initialize
         %r{(app|vendor)/(assets|javascript)/\w+/(.+\.(css|js|html|png|jpg|ts|jsx)).*} => :always
       }
       @default_patterns_changed = false
+
+      @ignore_patterns = [
+        %r{node_modules/}
+      ]
+      @default_ignore_patterns_changed = false
```

Comment:
Personally, I would keep this simple and backwards compatible with RLR as it exists today, and not by default ignore `node_modules` which may break some people's existing expectations for their project (especially not without updating documentation to indicate that node_modules is ignored). Rather, I think modifying the generator as you've done is plenty.

It also lets you simplify things by having `@ignore_patterns = []` in the initializer without needing the `@default_ignore_patterns_changed` state.

Reply:
@dymk 
Thank you for your feedback! 🙏
I’ve made the suggested changes in 3c5541697b7a4640e31f94b1cc3b1b08be35db34 :

- Updated the implementation to avoid ignoring `node_modules/` by default, maintaining compatibility with existing projects.
- Simplified the initializer by removing the `@default_ignore_patterns_changed` state.

Once the review is approved, I’ll rebase the fixup commit to keep the history clean 😄